### ISSUE TITLE
configure fails if LoopTools is compiled with quadruple precision

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#38]
The quadruple library is called `libooptools-quad.a` but configure checks only for `libooptools.a`. Do you think it makes sense to check for both?

**Comment by Expander** [comment]
I think at the moment there is no need to check for `libooptools-quad.a`, because we don't use it. (Also, it would not make much sense to use it, because we would not gain anything from quad precision as FlexibleSUSY does not provide quad precision routines.)

**Comment by wkotlarski** [comment]
Imagine the situation that you evaluate a loop functions close to some weird threshold. It could be beneficial to do the reduction with quadruple precision, even if the final result will be truncated to double precision. It's purely for test/validation purposes.

**Comment by Expander** [comment]
I agree that it would be good for testing.

However, I think we would not gain anything from using quad precision, because the *input* would still be double precision. So, the threshold would be defined in terms of the double precision input and I cannot imagine how the internals of LoopTools could "recover" the missing bits.

But maybe I'm overlooking something? Do you have a specific threshold in mind?

**Comment by Expander** [comment]
Could it be, that you need to compile FS with `-DQUAD=1` or in order to make it work? I'm asking, because in `clooptools.h` I see

~~~
#if QUAD
[...]
#else 
[...]
#endif 
~~~

So, depending on the value of `QUAD` the function signature of the LoopTools functions changes. Maybe this is a problem?

In the meta code we set ``SARAH`A0[0] -> 0`` (see `meta/CConversion.m`). However, it may be that there is a call to `A0(m)` and `m` becomes numerically small for your point.

**Comment by wkotlarski** [comment]
Great idea. Unfortunately, this doesn't help. Looking again at the error message, there's this
```
m =  7.029637473326906739105359474889669E-4940
```
Could it be that for whatever reason ``SARAH`A0[0] -> 0`` replacement doesn't work as intended? I'm testing everything with the SM, so the only particles that could potentially be massless are neutrinos and Goldstones.

**Comment by Expander** [comment]
Indeed, in the current implementation of the SM the neutrinos are massless. They contribute in the self-energy of the Z boson, for example, in form of a `H(p^2, mv^2, mv^2, Q^2)` loop function. `H0` is expressed in terms of `A0`, `B0` functions. So, indeed, the error may come from there.

In FS we set `A0(m^2) = 0` if `m` is below a small threshold (currently `1e-4`). This is reasonable, because in DREG `A0(0) = 0` by definition.

Could it be that either
* the neutrino masses are "not small enough" for LoopTools to fall below the threshold
* LoopTools does not set `A0(0) = 0` anyway
